swarm/api/http/sctx.go

```diff
@@ -0,0 +1,57 @@
+package http
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/swarm/api"
+	"github.com/ethereum/go-ethereum/swarm/sctx"
+)
+
+type contextKey int
+
+const (
+	uriKey contextKey = iota
+	requestErrorKey
+	requestCodeKey
+	decryptedKey
+	resolvedContentKey
+)
+
+func GetRUID(ctx context.Context) string {
```

Comment:
Maybe this functions should not be exported as they are used only by http package?